vault/src/execution.rs

```diff
@@ -429,14 +429,15 @@ pub async fn execute_open_requests(
     num_confirmations: u32,
     payment_margin: Duration,
     auto_rbf: bool,
+    no_execute_all: bool,
```

Comment:
I'd prefer to not have negations inside bool names (in cli args it's ok, but in internal code I'd prefer to avoid)